src/pages/pretest-checks/pretest-checks.ts

```diff
@@ -50,4 +49,8 @@ export class PretestChecksPage {
 
     eyesightFaultRecordingModal.present();
   }
+
+  gotoDL25() {
+    this.navCtrl.push(AllOnOnePage, {}, { animate: false });
```

Comment:
how comes no animation?

Reply:
I had to disable animation due UI glitch that was happening due replacing header